server/api/endpoint/ApiEndPoint.ts

```diff
@@ -0,0 +1,113 @@
+export const ApiEndPoint = {
+  Assessment: {
+    createEmail: '/api/assessment/:countryIso',
+    export: '/api/assessment/admin/export',
+  },
+
+  Audit: {
+    getFeed: '/api/audit/getAuditFeed/:countryIso',
```

Comment:
they must be method because there are parameters.
e.g. 
```
getFeed:(countryIso:string)=> joinPaths('audit','getAuditFeed', countryIso),
```
Then server side it's invoked as:
```
ApiEndpoint.Audit.getFeed(':countryIso')
```
And client side:
```
ApiEndpoint.Audit.getFeed('ITA')
```